src/Symfony/Component/ExpressionLanguage/composer.json

```diff
@@ -16,7 +16,8 @@
         }
     ],
     "require": {
-        "php": ">=5.5.9"
+        "php": ">=5.5.9",
+        "symfony/cache": "~3.1"
```

Comment:
@stof I know that `Symfony\Component\Cache\Adapter\ArrayAdapter` is used by default in `ExpressionLanguage.php`, and `Symfony\Component\Cache\CacheItem` is used in `ParserCache\ParserCacheAdapter.php`.
But honestly, I 've added this dependency mainly to make Travis CI PHP:7 builds passe.

You think I should remove it ?